lib/writeLog.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var config = require('../config/secrets');
+var logger = require('./logger');
+
+var logFlag = config.get().logger.api.log_user_activity;
```

Comment:
What if we just log default values for e.g. `userID`. Not sure if that's better or worse. What do you all think? @andresfib